src/modules/movies.js

```diff
@@ -0,0 +1,17 @@
+const createApp = async () => {
+  const key = localStorage.getItem('apiKey');
+  if (key === null || key === undefined || key === '') {
+    const rawResponse = await fetch('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/', {
+      method: 'POST',
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
+      },
+      // body: JSON.stringify({a: 1, b: 'Textual content'})
```

Comment:
- [ ]  You should remove un necessary comment in you project codes.  This will  make  your project  to have good clean codes.